labels.yaml

```diff
@@ -0,0 +1,282 @@
+- color: e11d21
```

Comment:
This is going to be a super nitpicky, dumb comment, but... 

1. Include a top level name (makes it much easier to amend this file format in the future if necessary)
2. start with name, not color.

e.g.

```
labels:
  - name: foo
    color: ...
  - name: bar
    color: ...
```